mlflow/genai/judges/optimizers/simba.py

```diff
@@ -24,6 +49,14 @@ class SIMBAAlignmentOptimizer(DSPyAlignmentOptimizer):
 
     Uses DSPy's SIMBA algorithm to optimize judge prompts through
     bootstrap aggregation with simplified parametrization.
+
+    Note on Logging:
+        By default, SIMBA optimization suppresses DSPy's verbose output.
+        To see detailed optimization progress from DSPy, set the MLflow logger to DEBUG::
+
+            import logging
+
+            logging.getLogger("mlflow.genai.judges.optimizers.simba").setLevel(logging.DEBUG)
```

Comment:
can we move this to the begining of the example?

Reply:
will do!